### ISSUE TITLE
Format Autocomplete Suggestions Vertically to make Keyboard navigation more clear

### ISSUE THREAD

**Issue by codeincarnate** [vector-im/element-web#17574]
### Problem

Currently auto-complete suggestions are displayed in a condensed list (as shown below). However, the suggestions can only be navigated on keyboard using the up-arrow and the down-arrow making for a confusing user experience since the left-arrow and right-arrow can't not be used to navigate the suggestions.

![Screen Shot 2021-06-07 at 4 06 05 PM](https://user-images.githubusercontent.com/199847/120989976-4c18e600-c7aa-11eb-8e43-ba2217856055.png)

### Potential Solution

A potential solution to this would be to adopt the approach used in Mattermost and Slack, which is to display the suggestions vertically with the ability to scroll through all of the options. In particular, the keyboard navigation hints which Slack has seems to present a good solution and good discoverability.

#### Mattermost

![Screen Shot 2021-06-07 at 3 46 32 PM](https://user-images.githubusercontent.com/199847/120989724-09570e00-c7aa-11eb-9d7e-7faaec088fcc.png)

#### Slack

<img width="473" alt="Screen Shot 2021-06-07 at 3 47 06 PM" src="https://user-images.githubusercontent.com/199847/120989708-06f4b400-c7aa-11eb-8cf2-5da29e89325d.png">

### Alternatives

Another alternative would be to format vertically and restrict the number of options displayed, which is done in Discord. However this seems somewhat undesirable as a user may not know the full name that they're using for an autocomplete suggestion.

Another potential option would be to make the suggestions navigable by using the left-arrow and right-arrow. Though as far as I know this has negative implications for accessibility.

**Comment by t3chguy** [comment]
> In particular, the keyboard navigation hints which Slack has seems to present a good solution and good discoverability.

This is actually great regardless of any other choices here

**Comment by HarHarLinks** [comment]
there is currently plenty space to put these hints
![image](https://user-images.githubusercontent.com/2803622/129764556-70e474dd-0995-4615-a1b1-d82f0c959840.png)

--- 

was there some change recently to navigating these? i have no idea right now (nightly 210817) how to select anything other than the first suggestion, but if muscle memory serves it used to be tab to select the next one.

**Comment by SimonBrandner** [comment]
@HarHarLinks, it's <kbd>Up</kbd> and <kbd>Down</kbd>
